### PR TITLE
Add Traditional Chinese (Taiwanese Chinese)

### PR DESCRIPTION
**Pull request by NeKoOuO**
Add Traditional Chinese(Taiwanese Mandarin)

### CODE REVIEW

**Comment by ShortDevelopment** [comment]
@NeKoOuO Please add yourself to [`CREDITS.md`](https://github.com/nearby-sharing/android/blob/main/CREDITS.md) by pushing a new commit onto this PR

**Comment by NeKoOuO** [comment]
done

File: src/Resources/values-tw/strings.xml [hunk 1]
```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+	<string name="app_name">就近共享</string>
+	<string name="app_titlebar_title_default">傳輸資料到您的 Windows 10 / 11 電腦</string>
+	<string name="app_titlebar_title_receive">從您的 Windows 10 / 11 電腦接收資料</string>
+	<string name="app_titlebar_title_receive_setup">設定與資訊</string>
+
+	<string name="assets_prefix">zh</string>
```
Comment (ShortDevelopment) [inline src/Resources/values-tw/strings.xml:8]: Please follow the [Wiki](https://github.com/nearby-sharing/android/wiki/Contributing#add-translation) to add the missing translations and set this `assets_prefix` to the correct language-prefix afterwards

**Review by ShortDevelopment — CHANGES_REQUESTED** [review]
@NeKoOuO The tag `tw` stands for the language [Twi](https://en.wikipedia.org/wiki/Twi)
Please get the correct one from the [iana language-subtag-registry](https://www.iana.org/assignments/language-subtag-registry/language-subtag-registry).

File: src/Resources/values-tw/strings.xml [hunk 2]
```diff
@@ -5,7 +5,7 @@
 	<string name="app_titlebar_title_receive">從您的 Windows 10 / 11 電腦接收資料</string>
 	<string name="app_titlebar_title_receive_setup">設定與資訊</string>
 
-	<string name="assets_prefix">zh</string>
+	<string name="assets_prefix">zh-tw</string>
```
Comment (ShortDevelopment) [inline src/Resources/values-tw/strings.xml:8]: Based on your description you might want to use `zh-Hant-TW`
> Type: redundant
Tag: zh-Hant-TW
Description: Taiwan Chinese in traditional script
Added: 2005-04-11

**Review by ShortDevelopment — CHANGES_REQUESTED** [review]

**Comment by NeKoOuO** [comment]
done

**Comment by ShortDevelopment** [comment]
You should also rename your files to match this new pre- / postfix.

**Comment by NeKoOuO** [comment]
Done!

**Comment by ShortDevelopment** [comment]
@NeKoOuO Now you just need to follow the rest of [this Wiki entry](https://github.com/nearby-sharing/android/wiki/Contributing#add-translation) to add the missing translation for the welcome and setup page

**Comment by NeKoOuO** [comment]
Done !

**Comment by ShortDevelopment** [comment]
Great, thank you very much!
I'll test this and then merge it afterwards!

**Review by ShortDevelopment — APPROVED** [review]

**Comment by NeKoOuO** [comment]
> Thank you very much for your contribution!
> I'm sorry this took so long; I've been quite busy over the last view months 🙈

Thanks 😌

**Pull request merged**